dbcon/mysql/ha_mcs_ddl.cpp

```diff
@@ -2320,7 +2325,10 @@ int ha_mcs_impl_create_(const char* name, TABLE* table_arg, HA_CREATE_INFO* crea
 
     int compressiontype = get_compression_type(thd);
 
-    if (compressiontype == 1) compressiontype = 2;
+    // MCOL-4685:
+    // remove the option to declare uncompressed columns (set infinidb_compression_type = 0).
+    if (compressiontype == 1 || compressiontype == 0)
```

Comment:
Changing this line we should also rename the first element of the enum [here](https://github.com/mariadb-corporation/mariadb-columnstore-engine/blob/develop/dbcon/mysql/ha_mcs_sysvars.cpp#L24).

Reply:
Could you rename the mentioned enum element to UNUSED plz?